docs/tutorials/best-practices/error-safety.md

```diff
@@ -128,7 +128,7 @@ if any.
 
 ```Lua hl_lines="17"
 local number = scope:Value(1)
-local double = scope:Computed(function(use)
+local double = scope:Computed(function(use, scope)
```

Comment:
Scope shouldn't be provided as it is never used
```suggestion
local double = scope:Computed(function(use)
```

Reply:
reverted